relayer/relays/beefy/parameters.go

```diff
@@ -89,13 +93,42 @@ func toBeefyClientCommitment(c *types.Commitment) *contracts.BeefyClientCommitme
 	}
 }
 
-func cleanSignature(input types.BeefySignature) (uint8, [32]byte, [32]byte) {
+func CleanSignature(input types.BeefySignature) (v uint8, r [32]byte, s [32]byte, reverted bool, err error) {
```

Comment:
Is there any link to documentation about this algorithm that we can add here? Maybe a link to the polkadot implementation?